src/Raven.Server/Documents/TransactionOperationsMerger.cs

```diff
@@ -580,6 +593,29 @@ private void MergeTransactionsOnce()
             }
         }
 
+        private void NotifyHighDirtyMemoryFailure(List<MergedTransactionCommand> pendingOps, string details, long transactionId)
+        {
+            var highDirtyMemory = new HighDirtyMemoryException(
```

Comment:
The exception is already there in the caller, why not just pass it in?

Reply:
done